rdmo/management/assets/js/components/element/Catalog.js

```diff
@@ -26,12 +26,17 @@ const Catalog = ({ config, catalog, elementActions, display='list',
 
   const toggleAvailable = () => elementActions.storeElement('catalogs', {...catalog, available: !catalog.available })
   const toggleLocked = () => elementActions.storeElement('catalogs', {...catalog, locked: !catalog.locked })
-
+  const addCurrentSite = () => elementActions.storeElement('catalogs', catalog, null, 'add-site')
+  const removeCurrentSite = () => elementActions.storeElement('catalogs', catalog, null, 'remove-site')
+  let has_current_site =  config.settings.multisite ? catalog.sites.includes(config.currentSite.id) : true
   const createSection = () => elementActions.createElement('sections', { catalog })
 
   const elementNode = (
     <div className="element">
       <div className="pull-right">
+        <ToggleCurrentSiteLink has_current_site={has_current_site}
```

Comment:
This needs to be hidden if `settings.MULTISITE is False`. (somehow it does not work as expected.)